README.md

```diff
@@ -41,7 +41,15 @@ $ make container IMAGE=your-repo/your-name:here
 To deploy your plugin image to an Ark server:
 
 1. Make sure your image is pushed to a registry that is accessible to your cluster's nodes.
-1. Run `ark plugin add <image>`, e.g. `ark plugin add gcr.io/heptio-images/ark-plugin-example`
+2. Run `ark plugin add <image>`, e.g. `ark plugin add gcr.io/heptio-images/ark-plugin-example`
+
+# Using the plugins
+
+A deployed plugin only means it's available to use, need to modify the configuration to make it effective, to do that:
```

Comment:
Edit to:
"When the plugin is deployed, it is only made available to use. To make the plugin effective, you must modify your configuration: